lib/ditty/controllers/application.rb

```diff
@@ -169,6 +174,21 @@ def view_location
       else
         content_type(:json) if request.accept.count.eql?(1) && request.accept.first.to_s.eql?('*/*')
       end
+      
+      return unless respond_to?(:browser) &&
+                    respond_to?(:request) &&
+                    respond_to?(:current_user) &&
+                    current_user.is_a?(Ditty::User) &&
+                    request.path_info != '/auth/identity/callback'
+
+      active_trait = current_user.user_login_traits.select { |t| t.active == true }.first
```

Comment:
Any way we can turn this feature on and off using a setting?

Reply:
No, I'll code it in.